src/com/example/Demo_Subj/Room.java

```diff
@@ -16,11 +16,15 @@
 
     private Bitmap bitmapRoom;
     private int roomID;
+    private int lowerRoomID;
+    private int upperRoomID;
+    private int leftRoomID;
+    private int rightRoomID;
     List<Integer> attachedRooms;
 
     private List<Item> itemList;
 
-    public Room (Bitmap bitmapRoomR, int ID, Context ctx) {
+    public Room (Bitmap bitmapRoomR, int ID, int leftRoom, int rightRoom, int upperRoom, int lowerRoom, Context ctx) {
```

Comment:
wäre es vllt schön die angeschlossenen räume direkt in ein AttachedRooms-objekt zu verpacken? In dem alle 4 richtungen gespeichert werden und außerdem eine funktion "isRoomAttached(int RoomID)" vorhanden ist? Das wäre hübsch und würde die Zuweisungsorgie weiter unten etwas verhindern.